src/coreclr/src/jit/gentree.h

```diff
@@ -7402,6 +7386,86 @@ inline unsigned int GenTree::GetRegSpillFlagByIdx(int regIndex) const
     return TYP_UNDEF;
 }
 
+//-----------------------------------------------------------------------------------
+// GetLastUseBit: Get the last use bit for regIndex
+//
+// Arguments:
+//     regIndex - the register index
+//
+// Return Value:
+//     The bit to set, clear or query for the last-use of the regIndex'th value.
+//
+// Notes:
+//     This must be a GenTreeLclVar or GenTreeCopyOrReload node.
+//
+inline unsigned int GenTree::GetLastUseBit(int regIndex)
+{
+    assert(regIndex < 4);
+    assert(OperIs(GT_LCL_VAR, GT_STORE_LCL_VAR, GT_COPY, GT_RELOAD));
+    static_assert_no_msg((1 << MULTIREG_LAST_USE_SHIFT) == GTF_VAR_MULTIREG_DEATH0);
+    return (1 << (MULTIREG_LAST_USE_SHIFT + regIndex));
+}
+
+//-----------------------------------------------------------------------------------
+// IsLastUse: Determine whether this node is a last use of the regIndex'th value
+//
+// Arguments:
+//     regIndex - the register index
+//
+// Return Value:
+//     true iff this is a last use.
+//
+// Notes:
+//     This must be a GenTreeLclVar or GenTreeCopyOrReload node.
+//
+inline bool GenTree::IsLastUse(int regIndex)
+{
+    assert(OperIs(GT_LCL_VAR, GT_STORE_LCL_VAR, GT_COPY, GT_RELOAD));
+    return (gtFlags & GetLastUseBit(regIndex)) != 0;
+}
+
+//-----------------------------------------------------------------------------------
+// IsLastUse: Determine whether this node is a last use of any value
+//
+// Return Value:
+//     true iff this has any last uses (i.e. at any index).
+//
+// Notes:
+//     This must be a GenTreeLclVar or GenTreeCopyOrReload node.
+//
+inline bool GenTree::HasLastUse()
+{
+    return (gtFlags & (GTF_VAR_DEATH_MASK)) != 0;
+}
+
+//-----------------------------------------------------------------------------------
+// SetLastUse: Set the last use bit for the given index
+//
+// Arguments:
+//     regIndex - the register index
+//
+// Notes:
+//     This must be a GenTreeLclVar or GenTreeCopyOrReload node.
+//
+inline void GenTree::SetLastUse(int regIndex)
+{
+    unsigned int bitToSet = gtFlags |= GetLastUseBit(regIndex);
```

Comment:
I realize you just copied this, but the `unsigned int bitToSet =` part is not used.

Reply:
Thanks - I'll fix that in a future change, to avoid another CI testing round.